includes/Core/Modules/Modules.php

```diff
@@ -183,9 +183,7 @@ public function __construct(
 			$this->core_modules[ Reader_Revenue_Manager::MODULE_SLUG ] = Reader_Revenue_Manager::class;
 		}
 
-		if ( Feature_Flags::enabled( 'signInWithGoogleModule' ) ) {
```

Comment:
This module should be added to the core modules list instead of defining it here: https://github.com/google/site-kit-wp/blob/64efe04702f71f8458b94d3c18f368081bfd164e/includes/Core/Modules/Modules.php#L147-L155

Reply:
Oh, right, thanks! 👍🏻